packages/dx-react-demos/src/theme-sources/material-ui/components/integer-type-provider.jsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import Input from 'material-ui/Input';
+import { withStyles } from 'material-ui/styles';
+import { DataTypeProvider } from '@devexpress/dx-react-grid';
+
+const styles = {
+  inputRoot: {
+    width: '100%',
+  },
+  numericInput: {
+    textAlign: 'right',
+  },
+};
+
+const IntegerEditorBase = ({ value, onValueChange, classes }) => (
+  <Input
+    type="number"
+    classes={{
+      root: classes.inputRoot,
+      input: classes.numericInput,
+    }}
+    value={value}
+    inputProps={{
+      step: 100,
```

Comment:
It looks like step is too big :)

Reply:
The 'amount' field value does not surpass the lower edge of 1000, so I thought it is OK to use 100 as a step. What value do you suggest instead?

Reply:
You can remove step at all.